backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/InvoiceCandDAO.java

```diff
@@ -2001,6 +2002,42 @@ private ICompositeQueryFilter<I_C_Invoice_Candidate> toFilter(@NonNull final Inv
 			filter.addFilter(invoiceCandidatesFilter);
 		}
 
+		final DocTypeId orderDocTypeId = query.getOrderDocTypeId();
```

Comment:
pls group together the C_Order sub-query to avoid multiple sub-queries about same table